src/AppInstallerCLICore/Core.cpp

```diff
@@ -1,9 +1,11 @@
-﻿// Copyright (c) Microsoft Corporation.
```

Comment:
What is going on here, has your editor injected a BOM?

Reply:
No. I had authentication issues submitting the PR from Visual Studio, yesterday. So I tried updating each file on github web. Looks like I mistakenly updated this file from the diff rather than from the file itself. Sorry about that.